templates/ts-apollo-mongodb-backend/src/custom-resolvers.ts

```diff
@@ -0,0 +1,25 @@
+import { GraphbackContext, GraphbackCRUDService, Maybe, Scalars } from "graphback";
+
+/**
+ * Types can be generated using GraphQL-Code-Generator
+ */
+export type Note = {
+  id?: Maybe<Scalars['String']>;
+  title?: Maybe<Scalars['String']>;
+  description?: Maybe<Scalars['String']>;
+};
+
+const getNoteService = (context: GraphbackContext) => {
+  return context.graphback?.services?.Note as GraphbackCRUDService<Note>
+}
+
+export const customResolvers = {
+  Query: {
+    printNote: async (parent: any, variables: any, context: GraphbackContext, info: any) => {
+      const noteService = getNoteService(context);
```

Comment:
How about a helper on the context to fetch a service?

```suggestion
      const noteService: GraphbackCRUDService<Note> = context.getService(Note)
```

Reply:
I feel like we are just patching the symptoms of a clunky API though.

Reply:
I was leading this https://graphql-modules.com/docs/introduction/dependency-injection#providers this morning and I like their API: Not to bring in a full fledged DI container but we kind of have it already and having something like `context.services.get(serviceName)` would be a big plus.

Reply:
That will be kinda cool, but you probably meant this?

```ts
context.service<User>("User")
````